mesh/mesh.hpp

```diff
@@ -1395,7 +1396,8 @@ class GeometricFactors
       DETERMINANTS = 1 << 2,
    };
 
-   GeometricFactors(const Mesh *mesh, const IntegrationRule &ir, int flags);
+   GeometricFactors(const Mesh *mesh, const IntegrationRule &ir, int flags,
+                    DofToQuad::Mode = DofToQuad::FULL);
```

Comment:
Same here, I think `DofToQuad::TENSOR` should be the default.

Reply:
`done`